cmd/ftl/main.go

```diff
@@ -82,6 +82,10 @@ func main() {
 	logger := log.Configure(os.Stderr, cli.LogConfig)
 	ctx = log.ContextWithLogger(ctx, logger)
 
+	err := projectconfig.CreateFilesIfNonexistent(ctx, cli.ConfigFlag)
```

Comment:
It would be better to only create the single file being written to (if it's being written to), rather than creating all referenced config files. We don't want to hide errors if the user accidentally fat fingers a config file under normal circumstances.

Reply:
Got it, that makes sense. Done! (assuming that's just the last path in the list, going off `LoadWritableConfig` for reference)